src/rpdk/core/contract/resource_client.py

```diff
@@ -188,6 +188,7 @@ def _update_schema(self, schema):
             {fragment_decode(prop, prefix="") for prop in identifier}
             for identifier in additional_identifiers
         ]
+        self.additional_identifiers_paths = self._additional_identifiers_paths
```

Comment:
you can just rename _additional_identifiers_paths to additional_identifiers_paths